go-fuzz/hub.go

```diff
@@ -116,6 +118,24 @@ func newHub(metadata MetaData) *Hub {
 			ro.intLits = append(ro.intLits, []byte(lit.Val))
 		}
 	}
+	if *flagDict != "" {
+		ro.strLits = nil // Discard existing tokens
+		fileName := *flagDict
+		log.Printf("reading token list from \"%v\"\n", fileName)
+		dictData, err := ioutil.ReadFile(fileName)
+		if err != nil {
+			log.Fatalf("could not read tokens from \"%v\": %v\n", fileName, err)
+		}
+		textData := string(dictData)
+		splits := strings.Split(textData,"\n")
```

Comment:
Not that it really matters, but let's not convert to string and then convert back to []byte. Use bytes.Split instead. Or use a bufio.Scanner, which'll be simpler yet.

Reply:
+1